### PR TITLE
feat(icon): add xs and s upload icon

### PR DESCRIPTION
**Pull request by mmmmmmaya**
Closes issue #400

### CODE REVIEW

File: packages/components/src/common/svgs/IconUploadSmall.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M7.00509 10.4791C6.77126 10.4815 6.54622 10.3873 6.38119 10.2175C6.21545 10.047 6.12446 9.81482 6.12937 9.5742L6.12942 4.23741L5.1555 5.37084C4.95072 5.61363 4.63628 5.72446 4.33084 5.6631C4.0248 5.60162 3.77422 5.37658 3.6755 5.0733C3.62464 4.91704 3.61821 4.75267 3.65256 4.59762C3.68508 4.45083 3.75414 4.3124 3.85664 4.19714L6.34778 1.29803C6.51188 1.10859 6.74815 1 6.99608 1C7.24401 1 7.48028 1.10859 7.64491 1.29864L10.1333 4.19457C10.3554 4.44158 10.4236 4.78984 10.3128 5.10109C10.2024 5.41133 9.93267 5.63268 9.61338 5.67641C9.32168 5.71644 9.02981 5.60239 8.83857 5.37308L7.86275 4.23741L7.86275 9.57012C7.86764 9.81161 7.77912 10.0409 7.6173 10.2109C7.48304 10.3519 7.3076 10.4424 7.11933 10.4701L7.00509 10.4791Z" fill="#3867FA"/>
```
Comment (masoudmanson) [inline packages/components/src/common/svgs/IconUploadSmall.svg:2]: I think we need to remove `fill="#3867FA"` attribute from path tags. Currently the icons won't change color and stay blue all the time!

File: packages/components/src/common/svgs/IconUploadSmall.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+<path d="M10.96 9.84674C10.96 9.36123 11.3255 8.96107 11.7963 8.90638L11.9067 8.90001C12.4296 8.90001 12.8535 9.32388 12.8535 9.84674L12.8515 12.0195L12.8436 12.1755C12.7889 12.6463 12.3888 13.0118 11.9033 13.0118H2.09673C1.57387 13.0118 1.15 12.5879 1.15019 12.0589L1.15 12V9.84674C1.15 9.32388 1.57387 8.90001 2.09673 8.90001C2.61959 8.90001 3.04346 9.32388 3.04346 9.84678L3.042 11.118H10.959L10.96 9.84674Z" fill="#3867FA"/>
```
Comment (masoudmanson) [inline packages/components/src/common/svgs/IconUploadSmall.svg:3]: Same as above.

**Review by masoudmanson — APPROVED** [review]
LGTM :D Thank you @mmmmmmaya 💯

**Review by clarsen-czi — APPROVED** [review]
Looks great @mmmmmmaya !! Thanks!

**Pull request merged**